Makefile

```diff
@@ -101,23 +124,23 @@ cluster-sync-resources:
 	fi
 
 
-cluster-sync-handler: $(local_handler_manifest)
-	IMAGE_REGISTRY=localhost:$(shell ./cluster/cli.sh ports registry | tr -d '\r') \
+cluster-sync-handler: $(local_handler_manifest) $(CLI) $(KUBECTL)
+	IMAGE_REGISTRY=localhost:$(shell $(CLI) ports registry | tr -d '\r') \
 		make push-handler
-	./cluster/cli.sh ssh node01 'sudo docker pull $(LOCAL_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)'
+	$(SSH) node01 'sudo docker pull $(LOCAL_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)'
 	# Temporary until image is updated with provisioner that sets this field
 	# This field is required by buildah tool
-	./cluster/cli.sh ssh node01 'sudo sysctl -w user.max_user_namespaces=1024'
+	$(SSH) node01 'sudo sysctl -w user.max_user_namespaces=1024'
 	$(KUBECTL) delete --ignore-not-found -f $(local_handler_manifest)
 	$(KUBECTL) create -f $(local_handler_manifest)
 
-cluster-sync-manager: cluster-sync-resources $(local_manager_manifest)
-	IMAGE_REGISTRY=localhost:$(shell ./cluster/cli.sh ports registry | tr -d '\r') \
+cluster-sync-manager: cluster-sync-resources $(CLI) $(local_manager_manifest)
+	IMAGE_REGISTRY=localhost:$(shell $(CLI) ports registry | tr -d '\r') \
 		make push-manager
-	./cluster/cli.sh ssh node01 'sudo docker pull $(LOCAL_REGISTRY)/$(MANAGER_IMAGE_FULL_NAME)'
+	$(SSH) node01 'sudo docker pull $(LOCAL_REGISTRY)/$(MANAGER_IMAGE_FULL_NAME)'
```

Comment:
why we need this?

Reply:
So we can change kubevirtci path just changing one line at the top of the Makefile.